### PR TITLE
[MMDS][CDAP-13128] UX fixes in experiment detailed view in MMDS

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Fixes model status pie chart to have fixed color scheme.
- Fixes various metrics bar charts in experiments detailed view to have fixed color scheme.
- Fixes directives and features popover based on design spec
- Allows user to copy directives to clipboard.
- Removes references to "deploy" in a model under an experiment.

JIRA:
- https://issues.cask.co/browse/CDAP-13128
- https://issues.cask.co/browse/CDAP-13384

### CODE REVIEW

File: cdap-ui/app/cdap/components/CopyableID/index.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ import {UncontrolledTooltip} from 'components/UncontrolledComponents';
 import Clipboard from 'clipboard';
 import IconSVG from 'components/IconSVG';
 import T from 'i18n-react';
+import uuid from 'uuid/v4';
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/CopyableID/index.js]: nit: seems like our current pattern is `import uuidV4 from 'uuid/v4';`

File: cdap-ui/app/cdap/components/CopyableID/index.js [hunk 2]
```diff
@@ -47,13 +50,27 @@ export default class CopyableID extends Component {
       showTooltip: !this.state.showTooltip
     });
   }
-  render() {
-    let idlabel = `A-${this.props.id}`;
-    if (this.props.idprefix) {
-      idlabel = `${this.props.idprefix}-${this.props.id}`;
+
+  renderToolTipText() {
+
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/CopyableID/index.js]: nit: extra line?

File: cdap-ui/app/cdap/components/Experiments/DetailedView/ModelStatusIndicator/index.js [hunk 3]
```diff
@@ -30,21 +31,40 @@ const DEFAULT_STATUS_MAP = {
...
-  TRAINING: {
-    className: 'spin fa-spin',
-    icon: 'icon-spinner'
+  [MODEL_STATUS.SPLITTING]: {
+    className: 'fa-spin ',
+    icon: 'icon-spinner',
+    color: MODEL_STATUS_TO_COLOR_MAP[MODEL_STATUS.SPLITTING]
+  },
+  [MODEL_STATUS.SPLIT_FAILED]: {
+    className: '',
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/Experiments/DetailedView/ModelStatusIndicator/index.js:40]: Do we need these empty strings for `className`? It's ok if it's undefined right, if we don't need to pass the prop?

Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/Experiments/DetailedView/ModelStatusIndicator/index.js:40]: I think its better to have `''` than undefined. `className` ignores `''` but renders a class class 'undefined' if its undefined.

Comment (tonybach) [reply cdap-ui/app/cdap/components/Experiments/DetailedView/ModelStatusIndicator/index.js:40]: Huh interesting, didn't know that

**Comment by tonybach** [comment]
Code LGTM :+1:

**Comment by ajainarayanan** [comment]
Demoed to @leacd . Merging.

**Pull request merged**